src/assets/scss/tokens/typography.scss

```diff
@@ -78,3 +77,7 @@
             "Segoe UI Emoji",
             "Segoe UI Symbol";
 }
+
+.win {
+    font-family: "Twemoji Country Flags";
```

Comment:
Do we really want to set this as the font family for the whole page?

Reply:
Now I've changed it to use only in the language selector